### ISSUE TITLE
Suggestion - formatNaValue	

### ISSUE THREAD

**Issue by BKeyport** [Tom-Hirschberger/MMM-ValuesByNotification#6]
When formatNaValue is set "false" and NA value is used, can we also remove valueUnit from the display? 

This will enable "Loading" type screens
"Temperature LOADING" 

Thx.

**Comment by Tom-Hirschberger** [comment]
Hi,

I think i can realize this very quickly by allowing to change the `valuePositions` with a threshold configuration.
You then can choose if you only want to display the naValue without the unit or even specify a icon that should be displayed instead.
I will check for a solution as fast as I can.

**Comment by Tom-Hirschberger** [comment]
Hi,

i released version 0.0.7 a few minutes ago which adds support of `valueNaPositions` and override of `valuePositions` with thresholds.
You new simply can configure something like:

```
		{
			module: "MMM-ValuesByNotification",
			position: "top_right",
			config: {
				updateInterval: 60,
				naValue: "Loading",
				valueNaPositions: "[v]",
				groups: [
					{
						items: [
							{
								notification: "TEST1",
								itemTitle: "Test 1",
								values: [
									{
										valueUnit: " &#176;C",
										valueFormat: "Number(${value}).toFixed(2)",
									},
								],
								// thresholds: [
								// 	{
								// 		type: "eq",
								// 		value: "Loading",
								// 		valuePositions: "[v]"
								// 	},
								// ]
							},
						]
					},
				]
			},
		},
```
